geode-core/src/main/java/org/apache/geode/management/internal/cli/result/model/CompositeResultModel.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.management.internal.cli.result.model;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import org.apache.geode.management.internal.cli.result.ResultData;
+
+public class CompositeResultModel extends AbstractResultModel {
+
+  private Map<String, SectionResultModel> content = new LinkedHashMap<>();
+
+  private int sectionCount = 0;
+
+  @Override
+  public String getType() {
+    return ResultData.TYPE_COMPOSITE;
+  }
+
+  public void setType(String type) {
```

Comment:
Are setType() and getType() really needed - setters/gettets without any corresponding field? Is this one of the areas that needs to be cleaned up after commands are converted to this model?

Reply:
That's right. Where necessary, Jackson will add type information in order to correctly de/serialize. This specific type info can then be removed.